lib/Driver/UnixToolChains.cpp

```diff
@@ -153,14 +159,13 @@ toolchains::GenericUnix::constructInvocation(const DynamicLinkJobAction &job,
 
   // Configure the toolchain.
   // By default, use the system clang++ to link.
```

Comment:
Comment needs correction (and would probably be a good place to explain why someone shouldn't just change it back to `clang++`).

Reply:
*sigh* that is what happens when you try to be quick.  Thanks for pointing that out.